### PR TITLE
config: add openshift 4.6

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

**Comment by stbenjam** [comment]
You'll want to run `make jobs` after you do that

**Comment by stbenjam** [comment]
@honza Also as part of this, could you add it to the [4.6 release controller](https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.6.json)?

See https://github.com/openshift/release/commit/f805f415e0449ca0d3b6397ff24390c18d9880ba as an example

**Comment by stbenjam** [comment]
This should be good to go -- we just need https://github.com/openshift/release/pull/9415#issuecomment-637758377

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      stream: nightly
+      version: "4.6"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-metal-ipi
+  cron: '* */4 * * *'
```
Comment (andfasano) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml:20]: This should be modified to `0 */4 * * *`

**Pull request merged**